apps/web/src/pages/ManagerDashboardPage/Components/PoolCandidateSearchRequestPreviewListItem.tsx

```diff
@@ -0,0 +1,157 @@
+import { IntlShape, useIntl } from "react-intl";
+import { useState } from "react";
+
+import {
+  FragmentType,
+  getFragment,
+  graphql,
+  PreviewListItemFragment,
+} from "@gc-digital-talent/graphql";
+import { commonMessages } from "@gc-digital-talent/i18n";
+import { PreviewList } from "@gc-digital-talent/ui";
+import { formatDate, parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+
+import { formatClassificationString } from "~/utils/poolUtils";
+
+import ReviewTalentRequestDialog from "./ReviewTalentRequestDialog";
+import { deriveChipSettings } from "../utils";
+
+const PreviewListItemPoolCandidateSearchRequest_Fragment = graphql(
+  /* GraphQL */ `
+    fragment PreviewListItem on PoolCandidateSearchRequest {
+      id
+      jobTitle
+      status {
+        value
+      }
+      requestedDate
+      applicantFilter {
+        qualifiedClassifications {
+          group
+          level
+        }
+      }
+      initialResultCount
+    }
+  `,
+);
+
+function buildTitle(request: PreviewListItemFragment, intl: IntlShape): string {
+  const classifications = unpackMaybes(
+    request.applicantFilter?.qualifiedClassifications,
+  );
+  const firstPart =
+    classifications.length == 1
+      ? formatClassificationString(classifications[0])
+      : null;
+
+  const secondPart =
+    request.jobTitle ?? intl.formatMessage(commonMessages.notProvided);
+
+  const completedTitle =
+    typeof firstPart === "string"
+      ? `${firstPart}${intl.formatMessage(commonMessages.dividingColon)}${secondPart}`
+      : secondPart;
```

Comment:
I'm not a big fan of `firstPart` and `secondPart`. I think just calling it what the variable would be a better option: `classification` and `requestJobTitle`.

Reply:
There's some room for improvement.  :sweat_smile: 
[improve variable names](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11653/commits/b15f6c24593293e81479afe992d072a972de1e68)